test/jdk/javax/swing/JColorChooser/Test6977726.java

```diff
@@ -21,22 +21,37 @@
  * questions.
  */
 
+import javax.swing.JColorChooser;
+import javax.swing.JLabel;
+
 /*
  * @test
  * @bug 6977726
- * @summary Checks if JColorChooser.setPreviewPanel removes the old one
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test6977726.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Checks if JColorChooser.setPreviewPanel(JLabel) doesn't remove the preview panel but
+ *          removes the content of the default preview panel
+ * @run main/manual Test6977726
  */
 
-import javax.swing.JApplet;
-import javax.swing.JColorChooser;
-import javax.swing.JLabel;
+public class Test6977726 {
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test6977726")
+                .instructions("Check that there is a panel with white text " +
+                        "titled \"Preview\" in the JColorChooser.")
```

Comment:
> Check that there is a panel with white text titled "Preview" in the JColorChooser.

I read it as if the text should be titled "Preview", although it should actually refer to the panel(maybe I'm wrong).

Since you modified the original "If you don't see the preview panel" by adding the "white text" check, you should check it on all platforms/LaFs that this text is really white and distinguishable from the background, otherwise it might come back to us one day with a failure of this test.

Reply:
When I wrote it I meant to say "Check that there is a panel (with white text) titled 'Preview'", but now that I read it again it is pretty unclear..
Does 
`Check that there is a panel with \"Text Preview Panel\" text and with title \"Preview\" in the JColorChooser.`
make more sense?